### PR TITLE
update Performance Max examples

### PR DESCRIPTION
**Pull request by friedenberg**

### CODE REVIEW

**Review by sundquist — DISMISSED** [review]
LGTM

File: examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs [hunk 1]
```diff
@@ -1010,8 +1012,17 @@ public string Next()
             {
                 AssetGroup = assetGroupResourceName,
 
-                // Unset for the root listing group filter node.
+                // Since this is the root node, do not set the ParentListingGroupFilter. For all
+                // other nodes, this would refer to the parent listing group filter resource name.
                 // ParentListingGroupFilter = "<PARENT FILTER NAME>"
+
+                // The Subdivision type means this node has children. This type is used for the root
```
Comment (jradcliff) [inline examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs:1019]: This mentions `Subdivision` but the enum used here is `UnitIncluded`.

**Review by jradcliff — APPROVED** [review]

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 2]
```diff
@@ -483,15 +485,14 @@ public MutateOperation CreateRoot()
...
         public void Run(
             GoogleAdsClient client,
             long customerId,
             long assetGroupId,
-            bool shouldReplaceExistingProductTree)
+            bool replaceExistingTree)
         {
-            // [START add_performance_max_product_listing_group_tree_1]
             GoogleAdsServiceClient googleAdsServiceClient =
                 client.GetService(Services.V10.GoogleAdsService);
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:498]: The comment starting on line 501 should be updated. That factory is specifically for create operations, and there is a separate factory for remove operations. In addition, you may want to consider swapping the instantiation of the createFactory with the replaceExistingTree block for a better flow. You don't actually use the create factory until after that block.

File: examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs [hunk 3]
```diff
@@ -629,8 +630,8 @@ public MutateOperation CreateRoot()
             MutateGoogleAdsResponse response = googleAdsServiceClient.Mutate(request);
 
             PrintResponseDetails(request, response);
-            // [END add_performance_max_product_listing_group_tree_1]
         }
+        // [END add_performance_max_product_listing_group_tree]
 
         // [START add_performance_max_product_listing_group_tree_7]
         /// <summary>
```
Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:637]: Nit in [this comment](https://github.com/googleads/google-ads-dotnet/blob/6f0678260aa7913865f1c0aab24bd61ccaef1d06/examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs#L694) - should be "removed and/or created".

Comment (devchas) [inline examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs:637]: [Here](https://github.com/googleads/google-ads-dotnet/blob/6f0678260aa7913865f1c0aab24bd61ccaef1d06/examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs#L704) shouldn't we be trimming "_RESULT" from the end?

**Pull request merged**